### PR TITLE
Consolidate similar meaning configuration keys properly inside another key

### PR DESCRIPTION
**Pull request by douglascamata**

### CODE REVIEW

File: pokecli.py [hunk 1]
```diff
@@ -182,10 +182,11 @@ def init_config():
     add_config(
         parser,
         load,
-        long_flag="--spin_forts",
+        long_flag="--spin",
```
Comment (TheSavior) [inline pokecli.py]: wouldn't this be --forts_spin based on your logic below?

Comment (douglascamata) [inline pokecli.py]: hum, let me try something

Comment (douglascamata) [inline pokecli.py]: @TheSavior if in `add_config` I rewrite the long_flag like this:

```
if embedded_in:
    long_flag = "{}_{}".format(embedded_in, long_flag)
```

What you think

Comment (TheSavior) [inline pokecli.py]: What if we scrap all of our cli arguments and instead dynamically let people override anything in the config.json, supporting deeply nested objects like so:

```
--username my_username --forts.avoid_circles false --location_cache true
```

Comment (DisasteR) [inline pokecli.py]: just go for long_flag="--forts_spin" for now.

rewriting all cli arguments management is a little out of this PR scope no ?

Comment (TheSavior) [inline pokecli.py]: Yeah, out of scope. I've been teasing @douglascamata about this change for a few days. It was a proposal from a few days ago that solved some very specific problems that we just never seemed to implement: https://github.com/PokemonGoF/PokemonGo-Bot/issues/573#issuecomment-234757535

**Comment by TheSavior** [comment]
:+1:

**Comment by TheSavior** [comment]
:+1:

**Pull request merged**